src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2666,6 +2666,13 @@ public abstract partial class Enum : System.ValueType, System.IComparable, Syste
     }
     public static partial class Environment
     {
+        public readonly struct ProcessCpuUsage
```

Comment:
If we think we might want to make it mutable in the future, we should avoid making it readonly now.

Reply:
I see no reason we need to make it mutable in the future. If you feel strongly about it I can apply the change but we may need to change that again after the deadline which will be API change that I want to avoid. Please let me know what do you think?

Reply:
I would keep it readonly. From .NET Framework Design Guidelines: "DO NOT define mutable value types.".

Reply:
I'm fine with it being immutable and leaving it as is. I'm just pointing out it's a breaking change to remove the readonly and add mutability, readonly buys us very little, and there's the discussion on the issue about it.